### PR TITLE
Exposed cartodb.js internals through mapDefinitionModel 

### PR DESCRIPTION
**Pull request by nobuti**
This PR implements #11846.

It relays on https://github.com/CartoDB/deep-insights.js/pull/534 and https://github.com/CartoDB/deep-insights.js/pull/534

### CODE REVIEW

File: lib/assets/core/javascripts/cartodb3/data/map-definition-model.js [hunk 1]
```diff
@@ -67,5 +67,35 @@ module.exports = Backbone.Model.extend({
...
+
+  getMapViewSize: function () {
+    return this._mapViewSize;
+  },
+
+  setStaticImageURLTemplate: function (staticImageURLTemplate) {
+    this._staticImageURLTemplate = staticImageURLTemplate;
+  },
+
+  staticImageURLTemplate: function () {
```
Comment (xavijam) [inline lib/assets/core/javascripts/cartodb3/data/map-definition-model.js]: `getStaticImageURLTemplate`?

File: lib/assets/core/javascripts/cartodb3/editor/editor-settings-view.js [hunk 2]
```diff
@@ -16,11 +16,12 @@ var REQUIRED_OPTS = [
...
   'settingsCollection',
   'visDefinitionModel',
-  'userModel'
+  'userModel',
+  'stateDefinitionModel'
 ];
 
 module.exports = CoreView.extend({
-  className: 'Editor-panel',
+  className: 'Editor-panel settings',
```
Comment (xavijam) [inline lib/assets/core/javascripts/cartodb3/editor/editor-settings-view.js]: `settings`?

**Comment by xavijam** [comment]
Checked in local and it is working :+1:, deploying in ded07.

**Comment by xavijam** [comment]
I've found something in the UI:
screen shot 2017-04-10 at 21 33 38

**Comment by nobuti** [comment]
Fixed.

**Pull request merged**